src/components/Dropdown/components/option/option.jsx

```diff
@@ -1,16 +1,24 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import React from "react";
 import { components } from "react-select";
+import TooltipWrapper from "../../../Tooltip/TooltipWrapper";
+import Tooltip from "../../../Tooltip/Tooltip";
 import "./option.scss";
 
-const Option = ({ Renderer, ...props }) => {
-  const { data } = props;
-  if (!Renderer) return null;
-
+const Option = ({ Renderer, data, ...props }) => {
   return (
-    <components.Option {...props} className="dropdown-wrapper__option--reset">
-      <Renderer {...data} />
-    </components.Option>
+    <TooltipWrapper
+      wrapWithTooltip={!!data?.tooltipProps}
+      tooltipProps={{ position: Tooltip.positions.RIGHT, ...data.tooltipProps }}
+    >
+      {Renderer ? (
+        <components.Option {...props} className="dropdown-wrapper__option--reset">
+          <Renderer {...data} />
+        </components.Option>
+      ) : (
+        <components.Option {...props} className="dropdown-wrapper__option--reset" />
```

Comment:
before the change it was null, why now it's changed?

Reply:
Responded in prev comment

Reply:
not a must but it can be more readable here by doing something like this instead of condition:
```
 <components.Option {...props} className="dropdown-wrapper__option--reset">
          {Renderer ? <Renderer {...data} /> : undefined}
</components.Option>

I did not use here Renderer && <Renderer {...data} /> because did not wanted to pass false as child
```

Reply:
Tried exactly what u r proposing, but it resulted in this
![image](https://user-images.githubusercontent.com/104433616/186154952-d1229670-df6a-4272-9da6-1213986d8f77.png)

And I'm not sure why, cause I thought that if u r passing `undefined` in place of `children` it should work the same as when u r not passing children at all. But here for some reason it works in a different way - maybe due to some `react-select` implementation details